packages/legend-query-builder/src/stores/fetch-structure/QueryBuilderFetchStructureState.ts

```diff
@@ -77,6 +78,14 @@ export class QueryBuilderFetchStructureState {
     this.implementation.initialize();
   }
 
+  initialize(): void {
+    if (this.implementation instanceof QueryBuilderTDSState) {
```

Comment:
isn't it better to make this a method on `QueryBuilderFetchStructureImplementation` and for GraphFetch, we do nothing?